test/functional/test_runner.py

```diff
@@ -555,14 +555,14 @@ def run_tests(*, test_list, src_dir, build_dir, tmpdir, jobs=1, enable_coverage=
     while i < test_count:
         if failfast and not all_passed:
             break
-        for test_result, testdir, stdout, stderr in job_queue.get_next():
+        for test_result, testdir, stdout, stderr, reason in job_queue.get_next():
             test_results.append(test_result)
             i += 1
             done_str = "{}/{} - {}{}{}".format(i, test_count, BOLD[1], test_result.name, BOLD[0])
             if test_result.status == "Passed":
                 logging.debug("%s passed, Duration: %s s" % (done_str, test_result.time))
             elif test_result.status == "Skipped":
-                logging.debug("%s skipped" % (done_str))
+                logging.debug("%s skipped (%s)" % (done_str, reason))
```

Comment:
```suggestion
                logging.debug(f"{done_str} skipped ({reason})")
```

nit: Could use f-strings? Also, could name this `skip_reason`?

Reply:
pushed